src/Microsoft.AspNetCore.Routing/DefaultParameterPolicyFactory.cs

```diff
@@ -2,49 +2,46 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using Microsoft.AspNetCore.Routing.Constraints;
+using Microsoft.AspNetCore.Routing.Patterns;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Options;
 
-namespace Microsoft.AspNetCore.Routing.Matching
+namespace Microsoft.AspNetCore.Routing
 {
-    internal class DefaultMatchProcessorFactory : MatchProcessorFactory
+    public class DefaultParameterPolicyFactory : ParameterPolicyFactory
```

Comment:
Can we keep the implementation of this internal? Why does this need to be made public. Apologies if I said somethign configusing.

Reply:
What would MVC use to resolve inline policies? Re-implement `DefaultParameterPolicyFactory`?

Reply:
It would resolve `ParameterPolicyFactory` from DI? I'm just talking about the **implementation**

Reply:
Of course. For a moment there I wasn't Thinking With DI.